_includes/section-title-box.html

```diff
@@ -2,7 +2,7 @@
 
 <header class="va-rr-sg-header va-rr-bg-{{templateclass}}">
   <h1 class="va-rr-sg-title">{{page.title}}</h1>
-  {% if page.description %}
-    <p class="va-rr-sg-description">{{page.description}}</p>
+  {% if page.description %}<br>
```

Comment:
Rather than using a `br` here, adjust the padding for `usa-content`. The easiest way is to use `.va-rr-sg-header .usa-content`.